content/md/assets/from-PAM-to-new-AM.md

```diff
@@ -0,0 +1,96 @@
+
+## Tags management
+The way we handled `tags` was not efficient enough.  
+Indeed, it was impossible for you to manage the list of tags created, and also impossible to delete tags.  
+
+That is why we replaced our old tag-management by the possibility to manage them with a `simple or multi-select` attribute type. This way, you have a dedicated screen to see all your tags, to translate them into different languages, and delete them if needed. It also avoids duplicates.  
+
+If you used tags to reference your product, you should be very happy to learn that, as it is now possible to add as many attributes as needed to your assets, you can easily create an asset attribute called `product reference` for example. You will be able to search for this value in the asset library.  
+
+:::info
+Thus, in the new asset manager, there is a dedicated tab in each asset, where you can list all the products linked to this asset. Very useful ;)
+:::
+
+:::tips
+You had asset categories and/or tags, and you wonder how you could retrieve them? We recommend you to read our [step-by-step guide](#) for the migration ;)
+:::
+
+## About the transformations
+We radically changed the way to define asset transformations. They are not applied per channel anymore, and you can use the reference file in your product sheets. You can also have up to 10 transformations per asset family. This way, you are able to define that, for your ecommerce channel, you want 3 different transformations.
+Each variation is created in a new attribute of the asset.
+
+Discover how new transformations work in the [dedicated article](#).
+
+# With a new asset manager come new powerful features
+Of course, we improved our existing asset management features. But we also added new ones, that very worth it. Let's discover them!
+
+## Automatic link between assets & products
+We know that most of you use a convention to name your assets, often containing the reference of the product, which makes it possible to define a rule to automate the link between assets and products.
+We really wanted to ease your daily work, that is why we worked on that specific feature in order to propose it natively in the PIM, that we call the `product link rule`. All the details are [here](#).
+
+## Enrich your assets with more attributes
+As mentioned above, you can add up to 100 attributes per asset family, in order to add marketing information to your assets. The list of available attributes is described in [this article](#).
+
+## All media can be managed in Akeneo, even those stored outside of the PIM
+Because we know that your assets can be stored on specific media servers, or in a DAM, we introduced a new attribute type `media link` in the Asset Manager, which enables you to add url link in your assets. This way, you can have in your PIM, videos stored on Youtube for example ;)
```

Comment:
to add a URL link to your assets. 

This way, the PIM can have links to videos stored on YouTube, for example.